networking/hardware_networks/configuring-sriov-rdma-cni.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="configuring-sriov-rdma-cni"]
+= Configuring an RDMA CNI for SR-IOV 
+include::_attributes/common-attributes.adoc[]
+:context: configuring-sriov-rdma-cni
+
+toc::[]
+
+Remote Direct Memory Access (RDMA) allows direct memory access between systems without involving either one's operating system.
+You can configure an RDMA CNI (Container Network Interface) on SR-IOV (Single Root I/O Virtualization) to enable high-performance, low-latency communication between containers.
+When you combine RDMA with SR-IOV, you provide a mechanism to efficiently expose RDMA capabilities to containers in Kubernetes or OpenShift clusters.
```

Comment:
maybe something like `you provide a mechanism to expose ethernet hw-containers for mellanox devices that can be used inside DPDK application.`